activestorage/test/controllers/representations/proxy_controller_test.rb

```diff
@@ -6,42 +6,37 @@
 class ActiveStorage::Representations::ProxyControllerWithVariantsTest < ActionDispatch::IntegrationTest
   setup do
```

Comment:
Since I'm technically changing the proxy controller with this solution, I believe we can change and improve all the tests in this file if you don't mind